tools/loop-back-lvm/entrypoint.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -euo pipefail
+
+cp ./create_lvm.sh /host/create_lvm.sh
+chroot /host ./create_lvm.sh
+rm /host/create_lvm.sh
+
+# let the monitoring script know we're done
+echo "done" >/ready
+echo "ready"
+while true; do
+  sleep 60
```

Comment:
FYI:  the script cannot receive signals when `sleep` is executing.  So, when docker/kubernetes tries to stop this container, it may hang for awhile and eventually get force killed

Reply:
So `sleep 1` is better IMO

Reply:
Sure.